src/main/java/CalculateSumHandler.java

```diff
@@ -0,0 +1,44 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.Future;
+
+public class CalculateSumHandler {
+    private static final int NUMBER_THREADS = 10;
```

Comment:
```suggestion
    private static final int THREADS_NUMBER = 10;
```
з точку зору англ так зрозуміліше буде

Reply:
Ok